people/s5ehfr9/page.en.md

```diff
@@ -3,4 +3,18 @@ name: s5ehfr9
 info:
     alias: Lucy
     location: Mianyang, Sichuan
----
\ No newline at end of file
+---
+
+## Description 
+
+She has been particularly interested in aerospace and military since childhood, and loves to play various flight and war simulation games. She also has a strong interest in game development, enjoys writing mods for the games she loves, and has even participated in the development of independent games. Besides, she is also an excellent drummer and has played in bands, and she is also good at English, painting and so on.  
+
+But it was her who looked excellent in everyone's eyes, the MtF thoughts that had been suppressed in her heart since she was a child exploded completely, and she suffered from severe depression from then on.  
```

Comment:
Although everyone think of her as an excellent person, she finally became unable to suppress the MtF thoughts that had been in her heart since she was a child, and she suffered from severe depression from then on.